### ISSUE TITLE
postgres-operator-ui:v1.12.0 exec format error on ARM64

### ISSUE THREAD

**Issue by acelinkio** [zalando/postgres-operator#2649]
Please, answer some short questions which should help us to understand your problem / question better?

- **Which image of the operator are you using?** 
ghcr.io/zalando/postgres-operator:v1.12.0
- **Where do you run it - cloud or metal? Kubernetes or OpenShift?** 
metal.  kubernetes. k3s .v1.29.2+k3s1 
- **Are you running Postgres Operator in production?**
no
- **Type of issue?** 
bug

On v1.12.0 of the UI, I am getting exec error on ARM based host.  I see there is an ARM based image published to ghcr.io but still getting this exec format error.  Did not have this error on v1.11.0 of the ui.  Also do NOT have this error with 1.12.0 of the postgres-operator. 

```log
exec /usr/local/bin/python: exec format error
```

**Comment by hemakshis** [comment]
Hi, I am facing this issue in my k8s cluster when upgrading helm version to v1.12.0 -

```
Failed to pull image "registry.opensource.zalan.do/acid/postgres-operator:v1.12.0": rpc error: code = NotFound desc = failed to pull and unpack image "registry.opensource.zalan.do/acid/postgres-operator:v1.12.0": failed to resolve reference "registry.opensource.zalan.do/acid/postgres-operator:v1.12.0": registry.opensource.zalan.do/acid/postgres-operator:v1.12.0: not found
```

This is how I am applying the helm chart -
```yaml
apiVersion: helm.toolkit.fluxcd.io/v2beta1
kind: HelmRelease
metadata:
  name: postgres-operator
  namespace: postgres-operator
spec:
  releaseName: postgres-operator
  chart:
    spec:
      chart: postgres-operator
      version: "1.12.0"
      sourceRef:
        kind: HelmRepository
        name: postgres-operator-charts
        namespace: postgres-operator
  interval: 10m
  values:
    installCRDs: true
    extraArgs: []
    image:
      registry: registry.opensource.zalan.do
      repository: acid/postgres-operator
      tag: v1.12.0
      pullPolicy: "IfNotPresent"
```

**Comment by goshkis** [comment]
I'm getting this error even on 1.11.0.
v1.29.4+k3s1 

Installed with helm
helm install postgres-operator-ui postgres-operator-ui-charts/postgres-operator-ui (...and with: --version 1.11.0)

Also tried -ui image from ghcr.io

**Comment by FxKu** [comment]
@goshkis @acelinkio Can you pull the `latest` image from master branch and try again?
Should be registry.opensource.zalan.do/acid/postgres-operator-ui:v1.12.0-2-g1f47f592

I want to do a patch release today and want to make sure this is fixed.

**Comment by crkochan** [comment]
I pulled `registry.opensource.zalan.do/acid/postgres-operator-ui:v1.12.0-2-g1f47f592` and tried to run it.

No joy I'm afraid.

```
exec /usr/local/bin/python: exec format error
```

**Comment by FxKu** [comment]
maybe it's because the `registry.opensorce...` image is not a multi-arch image and can only run on amd64.
@crkochan can you share your exec command?

**Comment by crkochan** [comment]
No exec command is being specified. By "pulled", I mean I patched the deployment object created by the postgres-operator-ui helm chart (v1.12.0, btw).

```
kubectl patch deployment postgres-operator-ui -p '{"spec":{"template":{"spec":{"containers":[{"name":"service", "image":"registry.opensource.zalan.do/acid/postgres-operator-ui:v1.12.0-2-g1f47f592"}]}}}}'
```

**Comment by FxKu** [comment]
We have released a fixed version of the 1.12 release today. I hope the issue is gone now with the 1.12.2 version. If not I would ask you to investigate and maybe come up with a PR as we don't get this error on our arm machines.

**Comment by acelinkio** [comment]
Confirmed that 1.12.2 is working on my arm hardware.  Image being used: `ghcr.io/zalando/postgres-operator-ui:v1.12.2`

closing ticket.  thanks @FxKu
